code/tests/utilities/orchestrator/test_prompt_flow.py

```diff
@@ -0,0 +1,127 @@
+from unittest.mock import ANY, AsyncMock, MagicMock, patch
+import json
+
+import pytest
+from backend.batch.utilities.orchestrator.prompt_flow import (
+    PromptFlowOrchestrator,
+)
+from backend.batch.utilities.parser.output_parser_tool import OutputParserTool
+
+@pytest.fixture(autouse=True)
+def llm_helper_mock():
+    with patch(
+        "backend.batch.utilities.orchestrator.prompt_flow.LLMHelper"
+    ) as mock:
+        llm_helper = mock.return_value
+
+        llm_helper.get_ml_client.return_value = MagicMock()
+        llm_helper.get_endpoint_name.return_value = "endpoint_name"
+        llm_helper.get_deployment_name.return_value = "deployment_name"
+        llm_helper.transform_chat_history_for_pf.return_value = []
+
+        yield llm_helper
+
+@pytest.fixture()
+def orchestrator():
+    with patch(
+        "backend.batch.utilities.orchestrator.prompt_flow.OrchestratorBase.__init__"
+    ):
+        orchestrator = PromptFlowOrchestrator()
+
+        orchestrator.config = MagicMock()
+        orchestrator.config.prompts.enable_content_safety = True
+
+        orchestrator.call_content_safety_input = MagicMock(return_value=None)
+        orchestrator.call_content_safety_output = MagicMock(return_value=None)
+
+        orchestrator.output_parser = OutputParserTool()
+
+        yield orchestrator
+
+def test_prompt_flow_init(orchestrator: PromptFlowOrchestrator):
+    assert orchestrator.ml_client is not None
```

Comment:
Minor, but could this assert that it's the same as the mock you're assigning on line 17?